CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# dbt_hubspot v0.19.1
+[PR #149](https://github.com/fivetran/dbt_hubspot/pull/149) includes the following updates:
```

Comment:
This PR actually doesn't include the relevant change. Request to update this to reference the PR which made the `quickstart.yml` change.
```suggestion
[PR #148](https://github.com/fivetran/dbt_hubspot/pull/148) includes the following updates:
```